### PR TITLE
Smugmug: fix album creation error (photo upload still isnt working)

### PR DESCRIPTION
**Pull request by seehamrun**
Album creation and response come back okay, but I'm currently getting an unauthorized error when trying to upload photos. Playing around with the smugmug upload api some more while I figure out what's going on. 

Weird because the response doesn't come back with any more information (like invalid signature or anything). 

#362

### CODE REVIEW

File: extensions/auth/portability-auth-smugmug/src/main/java/org/dataportabilityproject/auth/smugmug/SmugMugAuth/SmugMugOauthInterface.java [hunk 1]
```diff
@@ -55,7 +55,8 @@ public String getAuthorizationUrl(TokenSecretAuthData token, String permission)
     OAuthService service =
         new ServiceBuilder().provider(SmugMugOauthApi.class).apiKey(apiKey).apiSecret(apiSecret).build();
     String authUrl = service.getAuthorizationUrl(new Token(token.getToken(), token.getSecret()));
-    return String.format("%s&perms=%s", authUrl, permission);
+    // Request for permissions and for Full access (private and public content)
+    return String.format("%s&Permissions=%s&Access=%s", authUrl, permission, "Full");
```
Comment (holachuy) [inline extensions/auth/portability-auth-smugmug/src/main/java/org/dataportabilityproject/auth/smugmug/SmugMugAuth/SmugMugOauthInterface.java]: Will authUrl alrready contain a starting parameter, eg. ?something=something that can be appended to or do we need to check here?

File: extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java [hunk 2]
```diff
@@ -16,42 +16,20 @@
...
-import org.dataportabilityproject.transfer.smugmug.photos.model.ImageUploadResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumInfoResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumsResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugUser;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugUserResponse;
+import com.google.common.graph.ImmutableGraph;
+import com.google.common.net.HttpHeaders;
+import org.dataportabilityproject.transfer.smugmug.photos.model.*;
```
Comment (holachuy) [inline extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java]: Does google format expand these out or use the wildcard shortcut?

File: extensions/auth/portability-auth-smugmug/src/main/java/org/dataportabilityproject/auth/smugmug/SmugMugAuth/SmugMugOauthInterface.java (hunk 1 above)
Comment (seehamrun) [reply extensions/auth/portability-auth-smugmug/src/main/java/org/dataportabilityproject/auth/smugmug/SmugMugAuth/SmugMugOauthInterface.java]: authUrl will already contain the oauth query params for the auth request and callback parameter

File: extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java (hunk 2 above)
Comment (seehamrun) [reply extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java]: I'm not sure - I've seen inconsistencies with running google-format on my mac and on my ubuntu workstation. Other examples are where java standard library imports go (on their own like in this PR, or mixed in with all the other imports.)

**Pull request merged**